pkg/controller/authentication.go

```diff
@@ -164,13 +175,53 @@ func (c *Controller) loginEndpoint(ctx iris.Context) {
 			return
 		}
 
+		ctx.SetCookie(&http.Cookie{
+			Name:     "M-Token",
+			Value:    token,
+			Path:     "/",
+			Domain:   c.configuration.APIDomainName,
+			Expires:  time.Now().Add(7 * 24 * time.Hour),
+			MaxAge:   0,
+			Secure:   true,
+			HttpOnly: true,
+			SameSite: http.SameSiteStrictMode,
+		})
+
 		ctx.JSON(map[string]interface{}{
-			"token": token,
 			"users": login.Users,
 		})
 	}
 }
 
+// Logout
+// @Summary Logout
+// @id logout
+// @tags Authentication
+// @description Removes the `HttpOnly` authentication cookie for the user. This request does require the authentication
+// @description cookie is present. It will return a `403` if the cookie is not present.
+// @Router /authentication/logout [get]
+// @Security ApiKeyAuth
+// @Success 200
+// @Failure 403 {object} ApiError Cookie was not present.
+func (c *Controller) logoutEndpoint(ctx iris.Context) {
+	if cookie := ctx.GetCookie("M-Token"); cookie == "" {
+		c.returnError(ctx, http.StatusForbidden, "authentication required")
+		return
+	}
+
+	ctx.SetCookie(&http.Cookie{
+		Name:     "M-Token",
+		Value:    "",
+		Path:     "/",
+		Domain:   c.configuration.APIDomainName,
+		Expires:  time.Now(),
+		MaxAge:   0,
+		Secure:   true,
+		HttpOnly: true,
+		SameSite: http.SameSiteStrictMode,
+	})
```

Comment:
This should use the all-in-one method previously mentioned; and should just pass an empty token.